### PR TITLE
fix: fix output-resource-path required validate

### PR DESCRIPTION
**Pull request by qiuxiaolian**
**What this PR does / why we need it**:

- output-resource-path field  need't required validate

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @ladjzero 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/workflow/component/stage/TemplateStage.js [hunk 1]
```diff
@@ -99,7 +99,9 @@ class TemplateStage extends React.Component {
                           tooltip={_.get(argDes, r.name)}
                           hasFeedback
                           required={r.name !== 'output-artifact-path'}
-                          validate={required}
+                          validate={
+                            r.name !== 'output-artifact-path' && required
+                          }
```
Comment (ladjzero) [inline web/src/components/workflow/component/stage/TemplateStage.js]: validate={false} 不会有问题么

Comment (qiuxiaolian) [reply web/src/components/workflow/component/stage/TemplateStage.js]: 没问题，但改一下更好

**Pull request merged**